About/About/who-are-we.md

```diff
@@ -1,7 +1,23 @@
 ## Who are we?
 
-CHAOSS is a Linux Foundation project focused on creating analytics and metrics to help define community health. Work in the CHAOSS Project community is organized in two committees:
+CHAOSS is a Linux Foundation project focused on creating analytics and metrics to help define community health. Work in the CHAOSS Project community is organized in two committees and four workgroups:
+
+### Committees
 
 - The [Metrics Committee](https://chaoss.community/metrics) works on establishing implementation-agnostic metrics for measuring community activity, contributions, and health.
 
 - The [Software Committee](https://chaoss.community/software) works on producing integrated, open source software for analyzing software development, and definition of standards and models used in that software in specific use cases.
+
+###Workgroups
+
+The workgroups
+
+ The goal of the workgroups is to refine the metrics and to work with software implementations. The workgroups are built around the four categories of metrics that CHAOSS has identified. The workgroups are:
+
+- Growth Maturity and Decline
+
+- Diversity and Inclusion
```

Comment:
Line 17 and 19, the GMD and D&I workgroup names, could link out to the repositories that exist.

Also D&I is more mature workgroup and I would list it first.

Reply:
Linking was the plan but I was planning on taking them to the website page for the work groups. page/pages didn't exist when I made the changes so I have left them without a link.

Reply:
No issues moving D&I to the front